index.html

```diff
@@ -49,8 +50,13 @@
   </head>
 
   <body>
-    <h1 id="header">Basics: Beat That! 🚀</h1>
+    <h1 id="header">💀 Basics: Beat That! 🤡</h1>
     <div id="container">
+      <p>Hello! Welcome to Beat That! Click submit to start the game.</p>
+      <p>
+        Create a two-digit number by selecting the order of your dice rolls.
+      </p>
+      <p>The player with the highest number wins! Good luck!</p>
```

Comment:
Nice attempt at editing the HTML and CSS, the game definitely feels more like your own now doesn't it? Hope to see more of this in blackjack. Good that you applied the feedback from SPS too to put a message when the game is loaded, now its clear what the user has to do